src/lib/WIFI/devWIFI.cpp

```diff
@@ -539,6 +539,22 @@ static void WebUpdateGetTarget(AsyncWebServerRequest *request)
   json["product_name"] = product_name;
   json["lua_name"] = device_name;
   json["reg_domain"] = FHSSgetRegulatoryDomain();
+  json["git-commit"] = commit;
+#if defined(TARGET_TX)
+  json["is-tx"] = true;
+#endif
+#if defined(TARGET_RX)
+  json["is-rx"] = true;
+#endif
+#if defined(RADIO_SX127X)
+  json["is-sx127x"] = true;
```

Comment:
Perhaps have radio-type as a string rather than as `is` for all the different types.

Reply:
I meant instead if `is-sz127x`, `is-lr1121`

Reply:
> I meant instead if `is-sz127x`, `is-lr1121`

Could you propose a solution so that I wouldn't have to guess what you have in mind?